File: src/main/java/corp/phonebook/data/dto/UserDTO.java
package corp.phonebook.data.dto;

import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import jakarta.validation.constraints.Size;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class UserDTO {
    @Size(min = 1, max = 255, message = "Name must be between 2 and 50 characters")
    private String name;

    @Size(min = 1, max = 255, message = "Name must be between 2 and 50 characters")
    private String email;
}

File: src/main/kotlin/corp/phonebook/data/dto/UserDTO.kt
package corp.phonebook.data.dto

import jakarta.validation.constraints.Size

data class UserDTO(
    @field:Size(min = 1, max = 255, message = "Name must be between 2 and 50 characters")
    val name: String?,

    @field:Size(min = 1, max = 255, message = "Name must be between 2 and 50 characters")
    val email: String?
)